Cargo.lock

```diff
@@ -175,7 +181,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "5180c5a20655b14a819b652fd2378fa5f1697b6c9ddad3e695c2f9cedf6df4e2"
 dependencies = [
  "backtrace-sys",
- "cfg-if",
+ "cfg-if 0.1.10",
```

Comment:
Do we really need specify `0.1.10` here?

Reply:
Nope. I didn't modify this line. I just add `flate2` in module, and build it. It can be reversed if needed.

Reply:
Just use flate2 v1.0.11 as
`flate2 = { version = "1.0.11", default-features = false, features = ["zlib"] }`.

Reply:
done

But of no use. I suspect that `cfg_if` crate has released [v1.0.0](https://docs.rs/cfg-if/1.0.0/cfg_if/), then when `Cargo.lock` need to modify the specific version will be shown.

Reply:
> done
> 
> But of no use. I suspect that `cfg_if` crate has released [v1.0.0](https://docs.rs/cfg-if/1.0.0/cfg_if/), then when `Cargo.lock` need to modify the specific version will be shown.

did you forget to update Cargo.lock? Changing the dependency version of flake2 is for preventing it from depending on cfg-If v1.0.0.

Reply:
done

The exact version is needed:

`flate2 = { version = "=1.0.11", default-features = false, features = ["zlib"] }`